engine/lib/generic_printer/generic_printer_template.ml

```diff
@@ -35,6 +35,11 @@ struct
       method borrow_kind_Mut _x1 = default_document_for "borrow_kind_Mut"
       method borrow_kind_Shared = default_document_for "borrow_kind_Shared"
       method borrow_kind_Unique = default_document_for "borrow_kind_Unique"
+      method cf_kind_BreakOnly = default_document_for "cf_kind_BreakOnly"
+
+      method cf_kind_BreakOrReturn =
+        default_document_for "cf_kind_BreakOrReturn"
+
```

Comment:
Is this a drive-by fix, how is it related to Quote?

Reply:
This is generated code. I used the tool to regenerate this and this was part of it. This is because we changed the AST as part of https://github.com/hacspec/hax/pull/988, and this file hasn't been regenerated since.